docs/userguide/boundaryanalysis/index.md

```diff
@@ -229,143 +238,153 @@ boundary rules.
 
 **In C++:**
 
+```c++
 void listWordBoundaries(const UnicodeString& s) {
-UErrorCode status = U_ZERO_ERROR;
-BreakIterator\* bi = BreakIterator::createWordInstance(Locale::getUS(), status);
-bi->setText(s);
-int32_t p = bi->first();
-while (p != BreakIterator::DONE) {
-printf("Boundary at position %d\\n", p);
-p = bi->next();
-}
-delete bi;
+    UErrorCode status = U_ZERO_ERROR;
+    BreakIterator* bi = BreakIterator::createWordInstance(Locale::getUS(), status);
+    bi->setText(s);
+    int32_t p = bi->first();
+    while (p != BreakIterator::DONE) {
+        printf("Boundary at position %d\n", p);
+        p = bi->next();
+    }
+    delete bi;
 }
+```
 
 **In C:**
 
-void listWordBoundaries(const UChar\* s,
-int32_t len) {
-UBreakIterator\* bi;
-int32_t p;
-UErrorCode err = U_ZERO_ERROR;
-bi = ubrk_open(UBRK_WORD, 0, s, len, &err);
-if (U_FAILURE(err)) return;
-p = ubrk_first(bi);
-while (p != UBRK_DONE) {
-printf("Boundary at position %d\\n", p);
-p = ubrk_next(bi);
-}
-ubrk_close(bi);
+```c
+void listWordBoundaries(const UChar* s, int32_t len) {
+    UBreakIterator* bi;
+    int32_t p;
+    UErrorCode err = U_ZERO_ERROR;
+    bi = ubrk_open(UBRK_WORD, 0, s, len, &err);
+    if (U_FAILURE(err)) return;
+    p = ubrk_first(bi);
+    while (p != UBRK_DONE) {
+        printf("Boundary at position %d\n", p);
+        p = ubrk_next(bi);
+    }
+    ubrk_close(bi);
 }
 
```

Comment:
Missing end code block (```)

Reply:
thanks. Fixed in 126bb5d941ce11105dc2068253373409ab0fa9a4